web-frontend/src/pages/TransferCoins/ReceiveCoins/GenerateLink.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  Button,
+  Identicon,
+  Input,
+  MassaToken,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  Selector,
+} from '@massalabs/react-ui-kit';
+import { formatStandard } from '../../../utils/MassaFormating';
+import CopyContent from './CopyContent';
+import { useState } from 'react';
+import { AccountObject } from '../../../models/AccountModel';
```

Comment:
```suggestion
import { useState } from 'react';
import { formatStandard } from '../../../utils/MassaFormating';
import CopyContent from './CopyContent';
import { AccountObject } from '../../../models/AccountModel';

import {
  Button,
  Identicon,
  Input,
  MassaToken,
  PopupModal,
  PopupModalContent,
  PopupModalHeader,
  Selector,
} from '@massalabs/react-ui-kit';
```

Just to improve code vision.